src/platforms/python/guides/pymongo/index.mdx

```diff
@@ -44,11 +44,21 @@ automatically.
 
 </Note>
 
-## Other MongoDB libraries
+## Behavior
+
+The following information about your MonogDB queries will be available to you on Sentry.io:
+
+- Performance traces for all MongoDB queries
+- Breadcrumbs for all MongoDB queryies
+- Personal identifiable information (PII) will be stripped from all MongoDB queries. (This was tested for PyMongo 4.2 and below, but "should" also be future proof)
```

Comment:
```suggestion
- Personal identifiable information (PII) will be stripped from all MongoDB queries if `send_default_pii` is disabled in the SDK. (This was tested for PyMongo 4.2 and below, but "should" also be future proof)
```

Or something similar. If I was reading the current version without knowing the implementation I'd expect PII to never be sent.